packages/web/src/components/FlowContextMenu/index.jsx

```diff
@@ -59,6 +60,7 @@ function ContextMenu(props) {
     await queryClient.invalidateQueries({
```

Comment:
Since appKey is optional, the invalidateQueries call could be exectued only when appKey exists.